ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -208,6 +196,25 @@ export default class DepositAll extends IronfishCommand {
       confirmedBalance = CurrencyUtils.decode(balanceResp.content.confirmed)
       unconfirmedBalance = CurrencyUtils.decode(balanceResp.content.unconfirmed)
       pendingBalance = CurrencyUtils.decode(balanceResp.content.pending)
+
+      if (fee == null) {
+        try {
+          const response = await this.client.estimateFee({
+            fromAccountName: accountName,
+            receives: [
+              {
+                publicAddress: bankDepositAddress,
+                amount: CurrencyUtils.encode(confirmedBalance),
+                memo: graffiti,
+              },
+            ],
+          })
+          fee = CurrencyUtils.decode(response.content.medium)
+        } catch {
+          fee = 1n
+        }
+      }
```

Comment:
We could move this out of the while loop -- `fee` will always be non-null after the first iteration.

Or we could update to get a new estimate on each loop

Reply:
check `flags.fee` instead. If user did not pass in fee value. We will get a new estimate fee on each loop.